keyboards/alectardy98/alpinenoon/keymaps/alec/keymap.c

```diff
@@ -0,0 +1,219 @@
+/* Copyright 2022 Alectardy98
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include QMK_KEYBOARD_H
+
+enum layer_names {
+    _BASE,
+    _NUM,
+    _FRONT,
+    _FN
+};
+
+enum custom_keycodes {
+    TEST = SAFE_RANGE,
+    VDRT,                   //Desktop Right "set to move right a space on mac"
+    VDLT,                   //Desktop Left "set to move left a space on mac"
+    VDUP,                   //V-Desktop Up "set to mission controll on mac"
+    VDDN,                   //Min All "set to show desktop on mac"
+    EXPL,                   //New Explorer On Windows "set to show launchpad on mac"
+    SNIP,                   //Snip Tool Windows "set to save picture of selected area on mac"
+    COPY,                   //Copy All
+    PAST,                   //Paste Over
+    TASK,                   //Task Manager
+    QUIT,                   //Close App
+    DZRO,                   //Double Zero
+    ATAB,                   //Alt + Tab "brew install alt-tab on mac"
+    LPRC,                   //The "(" Key
+    RPRC,                   //The ")" Key
+    //Discord Section
+    THIS,                   //Macro for ":this:" emoji on discord
+    OOF,                    //Macro for ":oof:" emoji on discord
+    PIKA,                   //Macro for ":surprised:" emoji on discord
+    SCAT,                   //Macro for ":smiley_cat_spanish:" emoji on discord
+    FCAT,                   //Macro for ":smiley_cat_fancy:" emoji on discord
+    HART,                   //Macro for ":heart:" emoji on discord
+    DROL,                   //Macro for ":drooling_face:" emoji on discord
+    MONY,                   //Macro for ":smiley_takemoney:" emoji on discord
+    FHAT,                   //Macro for "::smiley_tip_fedora:" emoji on discord
+    SPIT,                   //Macro for ":smiley_spit_surprised:" emoji on discord
+    KING,                   //Macro for ":pepe_king_royal:" emoji on discord
+
+};
+    
+bool process_record_user(uint16_t keycode, keyrecord_t *record) {
+    if (record->event.pressed) {             //Code for macros
+        switch(keycode) {
+            case TEST:
+            if (record->event.pressed) {
+                SEND_STRING("TEST");
+                } else {}break;
```

Comment:
All of these should be changed to 
```suggestion
                }
                break;
```

Reply:
fixed